kafka_test_teardown.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Setup Kafka and create test topics
+
+set -ex
+
+echo "Stopping Kafka broker"
+kafka/bin/kafka-server-stop.sh
+echo "Stopping zookeeper"
+kafka/bin/zookeeper-server-stop.sh
```

Comment:
I am just wondering if can be done in the spec lifecycle we won't need to change anything in the travis.yml.